src/views/SettingsPage.vue

```diff
@@ -159,6 +215,27 @@ export default {
             alert(error);
           });
       }
+    },
+    addNewUser() {
+      const vm = this;
+      var user = new AV.User();
+      user.setUsername(vm.pendingChanges.newEmailAddress);
+      user.setPassword(vm.pendingChanges.newUserPassword);
+      user.setEmail(vm.pendingChanges.newEmailAddress);
+      user.setMobilePhoneNumber(vm.pendingChanges.newPhoneNumber);
+      user.set("fullName", vm.pendingChanges.newFullName);
+      user.signUp().then(
+        function() {
+          alert(
+            "User Created with email address: " +
+              vm.pendingChanges.newEmailAddress +
+              "."
```

Comment:
It looks cleaner if you say:

`` `User Created with email address: ${vm.pendingChanges.newEmailAddress}.` ``